lecture_03_functional_programming/hw/task03/task03.py

```diff
@@ -0,0 +1,51 @@
+# I decided to write a code that generates data filtering object from a list of keyword parameters:
+
+class Filter():
```

Comment:
Remove redundant parentheses here

```suggestion
class Filter:
```